test/extensions/transport_sockets/tls/ocsp/test_data/certs.sh

```diff
@@ -4,23 +4,21 @@
 
 set -e
 
+readonly DEFAULT_VALIDITY_DAYS=${DEFAULT_VALIDITY_DAYS:-730}
+readonly HERE=$(cd "$(dirname "$0")" && pwd)
+
+cd "$HERE" || exit 1
 trap cleanup EXIT
+
 cleanup() {
-  rm -f ./*_index*
-  rm -f ./*.csr
-  rm -f ./*.cnf
-  rm -f ./*_serial*
+    rm -f ./*.cnf
```

Comment:
nit: indentation

Reply:
That's coming from upstream, I'd rather keep it (to avoid merge conflicts in the future).

Reply:
makes sense. also, feel free to ignore these formatting nits, let me know if I should just ignore formatting in reviews (it's habitual for me to comment about it, but I can see that it might not always add value with what we're doing in this repo).

Reply:
Nah, thanks for catching these; Most of them were tabs injected by vim, good to get rid of those.